src/main/java/uk/gov/companieshouse/api/accounts/validation/FixedAssetsInvestmentsValidator.java

```diff
@@ -12,9 +12,10 @@
 import uk.gov.companieshouse.api.accounts.service.impl.CurrentPeriodService;
 import uk.gov.companieshouse.api.accounts.service.impl.PreviousPeriodService;
 import uk.gov.companieshouse.api.accounts.service.response.ResponseObject;
+import uk.gov.companieshouse.api.model.transaction.Transaction;
 
 @Component
-public class FixedAssetsInvestmentsValidator extends BaseValidator {
+public class FixedAssetsInvestmentsValidator extends BaseValidator implements NoteValidator<FixedAssetsInvestments> {
```

Comment:
Why are we refactoring Fixed AssetsInvestments ? I thought we are just going to do the Off balance sheet and other notes can be refactored later.

Reply:
This was my bad, we needed at least one concrete implementation of the `NoteValidator` interface else Spring errored on application start-up with an unsatisfied dependency exception in the `NoteValidatorFactory`. Minor annoyance, though I've ensured with the test team that we have adequate regression to cover fixed assets investments in the automation tests.

Reply:
Cool if we make sure to add this to the story as a comment that would be great . Cheers

Reply:
👍